security-whitepaper/cloud-native-security-whitepaper.md

```diff
@@ -267,49 +267,49 @@ Continuous Integration (CI) servers should be isolated and restricted to project
 
 #### Image hardening
 
-Container images constitute the first level of output from the build pipeline. As such, they must include security hardening that takes into consideration the threats to be mitigated while allowing some just-in-time configurations at the runtime phase to fit in a larger part of the ecosystem.
+Container images constitute the first level of output from the build pipeline. As such, they must include security hardening that takes into consideration the threats to be mitigated while allowing some just-in-time configurations at the runtime phase to facilitate integration with the broader ecosystem.
 
-In respect of the [security assurance objectives](#security-assurance), the following questions should be evaluated:
+With respect to the [security assurance objectives](#security-assurance), the following questions should be evaluated:
 
 - Should the execution environment be restricted to a specific user?
 - Should the resource's access be limited?
 - Should process execution be restricted at kernel level?
 
 #### Container Application Manifest Scanning
 
-Application manifests describe the configurations required for the deployment of containerized applications. As mentioned in the [Security Benchmarks](#security-benchmarks) section, guides and recommendations such as the [NIST 800-190](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-190.pdf) publication recommend best practice security practices and configurations for application containers. Consequently, it is vital to use tools to scan these application manifests in the CI/CD pipeline in order to identify configurations that could potentially result in an insecure deployment posture.
+Application manifests describe the configurations required for the deployment of containerized applications. As mentioned in the [Security Benchmarks](#security-benchmarks) section, guides such as the [NIST 800-190](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-190.pdf) publication offer recommended security practices and configurations for application containers. It is vital to scan application manifests in order to identify configurations that could potentially result in an insecure deployment posture.
```

Comment:
It is important to call out scanning of manifests in the CI/CD pipeline.  Simply scanning manifests outside of this breaks potential policy enforcement rules that would stop non-compliant or insecure artifacts from being delivered.